### PR TITLE
PE-3039 - Fixes getSandboxSubdomain 

### PR DESCRIPTION
**Pull request by matibat**
Changes

- Adds failing unit tests for the already existent testable methods
  - ✅ Manually checked that the expected value in the tests matches the real behavior on GWs.
- Replace the current implementation with the utils in the unit tests
  - ✅ Manually checked the implementation against the FlutterApp
  - Unit test each individual method

### CODE REVIEW

File: test/api/sandbox_test.dart [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+//       'cSJLdZFchg7iMmMXutn3D6dXuCHOCh4_vr6CmfGDsCc',
+//   'vxxdwy6scapybz7ekkw32ua5naseioagnu4uutbq3cftvivl':
+//       're47Y9IQH4-Dn5FKtvVAdaCRE-OAZtOUpMMNiLOqKrw',
+//   'bujjxactpuy7cmm3ul6fjuwuqnbymloddqvsjswhxsvkd255bq':
+//       'DRKbgFN9MfExm6L8VNLUg0OGLcMcKyTKx7yq_oeu9DM',
+//   'b4zy2sx2upeczfaxpscxfuglgh2mljo4gu6ycuhujgz5yjbc':
+//       'DzONSvqjyCyUF3yFctDLMfTFpd_w-1PYFQ9Emz3CQic',
+//   'kruhyqr3j24gxknxda2pbqa6ru2ctocctwr4z3hesnjukreb':
+//       'VGh8QjtOuGuptxg08MAejTQpuEKdo8zs_5JNTRUSB_Q',
+// };
```
Comment (matibat) [inline test/api/sandbox_test.dart:28]: Multiple test cases vs one single case?

**Review by matibat — COMMENTED** [review]
Self-review check point.

File: test/api/sandbox_test.dart (hunk 1 above)
Comment (karlprieb) [reply test/api/sandbox_test.dart:28]: you can keep multiple as you have them already, but they will not bring too much value

Comment (matibat) [reply test/api/sandbox_test.dart]: I simply took the TX IDs already present in there to use as input for the method.
29bfd003bca6dee8083ce6fab945f9c4edc0f8f1

**Review by karlprieb — CHANGES_REQUESTED** [review]
Why do we have `test/api/sandbox_test_utils.dart`? I get that you were comparing the new functions with the old ones, but now we only have the new ones and they're tested, we don't need to compare anymore.
We don't have unit tests for `nextMultipleOf4`

**Pull request merged**